sympy/ntheory/factor_.py

```diff
@@ -2040,3 +2040,79 @@ def eval(cls, n):
                 raise ValueError("n must be a positive integer")
             else:
                 return sum(factorint(n).values())
+
+
+MERSENNE_PRIME_EXPONENTS = (2, 3, 5, 7, 13, 17, 19, 31, 61, 89, 107, 127, 521, 607, 1279, 2203, 2281, 3217, 4253, 4423, 9689, 9941, 11213, 19937, 21701, 23209, 44497, 86243, 110503, 132049, 216091, 756839, 859433, 1257787, 1398269, 2976221, 3021377, 6972593, 13466917, 20996011, 24036583, 25964951, 30402457, 32582657, 37156667, 42643801, 43112609, 57885161, 74207281, 77232917, 82589933)
+
+def mersenne_prime_exponents(nth):
+    """
+    It returns the nth Mersenne Prime Exponents.
+    Examples
+    ========
+
+    >>> from sympy.ntheory.factor_ import mersenne_prime_exponents
+    >>> mersenne_prime_exponents(1)
+    2
+    >>> mersenne_prime_exponents(20)
+    4423
+    """
+    n = as_int(nth)
+    if n < 1:
+        raise ValueError("nth must be a positive integer; prime(1) == 2")
+    if n > 51:
+        raise ValueError("There are only 51 perfect numbers; nth must be less than or equal to 51")
+    return MERSENNE_PRIME_EXPONENTS[n-1]
+
+
+def ischeck_perfect_number(n):
```

Comment:
How about `is_perfect(n)`?

Reply:
okay, will change it

Reply:
i have changed `ischeck_perfect_number` to `is_perfect` and `ischeck_mersenne_prime` to `is_mersenne_prime`